afs-core/src/main/java/com/powsybl/afs/AbstractNodeBase.java

```diff
@@ -175,16 +180,44 @@ private boolean nodeNameAlreadyExists(String name) {
         return childNodes.stream().filter(nodeInfo -> !nodeInfo.getId().equals(getId())).anyMatch(nodeInfo -> nodeInfo.getName().equals(name));
     }
 
-    public void archive(Path dir) {
+    public void archive(Path dir, boolean useZip) throws IOException {
```

Comment:
Don't declare IOException, but wrap it in UncheckIoException (as it was done before)